Assets/MRTK/Core/Services/BaseBoundarySystem.cs

```diff
@@ -36,6 +54,8 @@ public abstract class BaseBoundarySystem : BaseCoreSystem, IMixedRealityBoundary
         /// <inheritdoc/>
         public override void Initialize()
         {
+            ReadProfile();
```

Comment:
It feels like this order is a bit off - ReadProfile needs BoundaryProfile (i.e. if it's null, it will crash right?)

Or is the check of BoundaryProfile == null on line 59 not needed?

Reply:
@davidkline-ms

Reply:
Let me double check. I believe all of the systems that have this method may behave the same.